wowchemy/assets/scss/wowchemy/elements/_callout.scss

```diff
@@ -2,6 +2,7 @@
 
 /* Style asides as Bootstrap alerts. */
 .article-style aside {
+  // stylelint-disable-next-line scss/at-extend-no-missing-placeholder
```

Comment:
I don't pretend to fully understand placeholders and extends in SCSS, but I believe the placeholder would have to be in the bootstrap vendored code which is a no-go. Thus we do a single-line disable.